shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -198,7 +198,7 @@ fn test_cassandra_peers_rewrite() {
             assert_query_result(
                 &rewrite_port_connection,
                 "SELECT native_port, native_port FROM system.peers_v2;",
-                &[&[ResultValue::Int(9044), ResultValue::Int(9042)]],
+                &[&[ResultValue::Int(9044), ResultValue::Int(9044)]],
```

Comment:
was this test previously wrong?

Reply:
or is this to catch the fixed subsequent case?

Reply:
The test was checked in with an incorrect assert so that we could demonstrate the incorrect behaviour that was present at the time.